src/AutoRest.CSharp.V3/Output/Models/Serialization/BinarySerialization.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using AutoRest.CSharp.V3.Generation.Types;
+
+namespace AutoRest.CSharp.V3.Output.Models.Serialization.Xml
+{
+    internal class BinarySerialization : ObjectSerialization
```

Comment:
Not sure I like this, binary is not a way to serialize objects, it's completely different kind of request body, multipart would eventually be another one.

Reply:
Fixed.